2025-03-08 14:12:39,313 INFO    MainThread:41111 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-08 14:12:39,313 INFO    MainThread:41111 [wandb_setup.py:_flush():67] Configure stats pid to 41111
2025-03-08 14:12:39,313 INFO    MainThread:41111 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/.config/wandb/settings
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_setup.py:_flush():67] Loading settings from /home/pallette/Documents/GitHub/Bachelor/wandb/settings
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250308_141239-z6oz539h/logs/debug.log
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/pallette/Documents/GitHub/Bachelor/wandb/run-20250308_141239-z6oz539h/logs/debug-internal.log
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:init():761] calling init triggers
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'model': 'UNet', 'dataset': 'cropped_objects', 'batch_size': 6, 'val_batch_size': 6, 'learning_rate': 0.01, 'global_image_size': (650, 650), 'num_epochs': 10, 'gaussian_blur': True, 'post_processing_threshold': 0.5, 'distance_threshold': 10, '_wandb': {}}
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:init():784] starting backend
2025-03-08 14:12:39,314 INFO    MainThread:41111 [wandb_init.py:init():788] sending inform_init request
2025-03-08 14:12:39,318 INFO    MainThread:41111 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-08 14:12:39,318 INFO    MainThread:41111 [wandb_init.py:init():803] backend started and connected
2025-03-08 14:12:39,319 INFO    MainThread:41111 [wandb_init.py:init():896] updated telemetry
2025-03-08 14:12:39,324 INFO    MainThread:41111 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-08 14:12:39,775 INFO    MainThread:41111 [wandb_init.py:init():995] starting run threads in backend
2025-03-08 14:12:39,844 INFO    MainThread:41111 [wandb_run.py:_console_start():2377] atexit reg
2025-03-08 14:12:39,845 INFO    MainThread:41111 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-08 14:12:39,845 INFO    MainThread:41111 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-08 14:12:39,845 INFO    MainThread:41111 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-08 14:12:39,847 INFO    MainThread:41111 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-08 14:12:44,906 WARNING MsgRouterThr:41111 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/pallette/Documents/GitHub/Bachelor/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-08 14:12:44,907 INFO    MsgRouterThr:41111 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
